Makefile

```diff
@@ -96,24 +97,28 @@ FILES_WITH_IMAGE:= .circleci/config.yml Dockerfile Dockerfile.echosrv \
 	Dockerfile.test Dockerfile.fcurl release/Dockerfile.in Webtest.sh
 # then run make update-build-image and check the diff, etc... see release/README.md
 update-build-image:
-	docker pull ubuntu:focal
+	docker buildx create --use
 	$(MAKE) docker-push-internal IMAGE=.build TAG=$(BUILD_IMAGE_TAG)
 
+SED:=sed
 update-build-image-tag:
-	@echo 'Need to use gnu sed (brew install gnu-sed; PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$$PATH")'
-	sed --in-place=.bak -e 's!$(DOCKER_PREFIX).build:v..!$(BUILD_IMAGE)!g' $(FILES_WITH_IMAGE)
+	@echo 'Need to use gnu sed (brew install gnu-sed; make update-build-image-tag SED=gsed)'
+	$(SED) --in-place=.bak -e 's!$(DOCKER_PREFIX).build:v..!$(BUILD_IMAGE)!g' $(FILES_WITH_IMAGE)
 
 docker-version:
 	@echo "### Docker is `which docker`"
 	@docker version
 
 docker-internal: dependencies
 	@echo "### Now building $(DOCKER_TAG)"
-	docker build -f Dockerfile$(IMAGE) -t $(DOCKER_TAG) .
+	# --load doesn't work on mac m1 with docker desktop even after docker buildx create --use - thoughts?
+	docker buildx build --platform $(BUILDX_PLATFORMS) -f Dockerfile$(IMAGE) --load -t $(DOCKER_TAG) .
```

Comment:
@aleskandro any idea about why I get 
```
error: docker exporter does not currently support exporting manifest lists
```
If I try to use --load ?
(mac m1 docker desktop)
```
$ docker buildx ls
NAME/NODE             DRIVER/ENDPOINT             STATUS  PLATFORMS
priceless_lovelace *  docker-container                    
  priceless_lovelace0 unix:///var/run/docker.sock running linux/arm64, linux/amd64, linux/amd64/v2, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/mips64le, linux/mips64, linux/arm/v7, linux/arm/v6
desktop-linux         docker                              
  desktop-linux       desktop-linux               running linux/arm64, linux/amd64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/arm/v7, linux/arm/v6
default               docker                              
  default             default                     running linux/arm64, linux/amd64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/arm/v7, linux/arm/v6
```

Reply:
Hello, yes, I don't remember at this time why I added the --load there, but it is not supported when multiple platforms are passed through --platform.

Do we need the images built by the docker-internal target immediately in the docker engine registry? If not, we can skip the --load as I did in the last commit (See https://github.com/docker/buildx/issues/59).

I think we shouldn't, as all the workflow should now be moved to docker buildx, we will use the buildx registry that supports manifest lists.

A bit worried, instead, about the buildx create command that you added.
Should we consider the builder is already created by a user that wants to build? Beyond the builder, based on the Host OS, you'd have more requirements.

E.g., on Fedora, with docker-ce:
```sh
# Install docker CE
dnf install -y yum-utils
yum-config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
dnf install -y docker-ce docker-ce-cli containerd.io 
gpasswd -a fedora docker
systemctl enable docker && systemctl start docker

# Enable multi-arch support and buildx
dnf install -y qemu-user-static
cat /proc/sys/fs/binfmt_misc/qemu-aarch64
docker buildx create --name mybuilder
docker buildx use mybuilder
```

Should we add some info like this in the readme?

Reply:
<edited the comment above>

Reply:
edited the comment above

Reply:
Probably could edit release/README.md with that info

At the moment I'm the one manually pushing the .build using the make targets but yes it's probably not correct to keep making a context (I was wondering if there is a sort of create or update context...)

Reply:
also it's odd I kept checking and didn't get notifications from github that there were updates on the mr

Reply:
we'll follow up in #551